libs/@guardian/source-react-components/src/choice-card/ChoiceCard.tsx

```diff
@@ -49,6 +55,10 @@ export interface ChoiceCardProps
 	 * The type of input you want
 	 */
 	type?: 'radio' | 'checkbox';
+	/**
+	 * A component level theme to override the colour palette of the button
+	 */
+	theme?: ChoiceCardTheme | 'light' | 'dark';
```

Comment:
I think i imagined we were going to treat any 'dark' theme as the just the same as any other theming option - if we're specifying it here then that gives it some kind of special status